### DISCUSSION TITLE
Migrate from Zipkin to Jaeger and use elasticsearch. All this in Rancher.

### DISCUSSION THREAD

**Discussion by Dubrovsky18** [jaegertracing/jaeger#5003] [Q&A]
Hello. Thanks in advance. 
I've operated Jaeger (All in one) in my system. In Load Balancing, I redirected all trafic from zipkin to Jaeger, which see only one service - jaeger-all-in-one. What I need to do, that Jaeger could find other named services and them trafic?
```
--es.username=xxxx --es.password=xxxx --es.server-urls=xxxxx:9200 --es.tls.enabled=true --es.tls.ca=xxxxx.pem --log-level=debug --metrics-http-route=/api/v1/spans --collector.zipkin.host-port=:9411
```
P.s. in env SPAN_STORAGE_TYPE=elasticsearch

**Comment by yurishkuro** [comment]
There is not enough information to troubleshoot your private setup. Please follow https://www.jaegertracing.io/docs/1.52/troubleshooting/ and if you exhaust the options mention there then post your findings in more details.

**Reply by Dubrovsky18** [reply]
![Jaeger](https://github.com/jaegertracing/jaeger/assets/89976250/e845113b-c0ce-49c2-9f3c-c7735178ed4a)
Situation.
I’ve already dealt with fixing mistakes. In request "/metrics"  we get the response
